Content.Client/Instruments/InstrumentSystem.cs

```diff
@@ -52,13 +52,44 @@ private void OnShutdown(EntityUid uid, InstrumentComponent component, ComponentS
         EndRenderer(uid, false, component);
     }
 
+    public void SetMaster(EntityUid uid, EntityUid? masterUid)
+    {
+        if (!TryComp(uid, out InstrumentComponent? instrument))
+            return;
+
+        RaiseNetworkEvent(new InstrumentSetMasterEvent(uid, masterUid));
+    }
+
+    public void SetFilteredChannel(EntityUid uid, int channel, bool value)
```

Comment:
`value` is the least descriptive name for a bool

Reply:
Given the name of the method, I think this is plenty descriptive.